pepper-apis/src/main/java/org/broadinstitute/ddp/export/DataExporter.java

```diff
@@ -183,11 +182,6 @@ public DataExporter(Config cfg) {
         componentNames.add("PHYSICIAN");
     }
 
-    @Override
-    public void close() throws IOException {
```

Comment:
Now that the ES client is a singleton, we can freely use a DataExporter object without having to call close.